# Calculates a path between two points
# Request:
#   start:
#     - Starting point on the grid (in meters)
#     - Z value is ignored
#     - In the grids parent frame
#   end:
#     - Ending point on the grid (in meters)
#     - Z value is ignored
#     - In the grids parent frame

geometry_msgs/Point start
geometry_msgs/Point end

---

# Response:
#   path:
#     - List of poses returned in the grid's parent frame
#     - Oriented facing towards the next point
#     - Empty if no path was found
#   path_sparse:
#     - List of poses returned in the grid's parent frame
#     - Oriented facing towards the next point
#     - Empty if no path was found
#     - Acts the same as "path" but removes intermediate steps if they form a straight line

geometry_msgs/PoseArray path
geometry_msgs/PoseArray path_sparse
